### ISSUE TITLE
Double Line Comment in Run Directive leads to `InvalidPython`

### ISSUE THREAD

**Issue by gordonkoehn** [snakemake/snakefmt#196]
It appears two single-line comments at the start of the run directive lead to a parsing error `Invalid Python` in the line following after. 

The rule runs fine. The error message disappears when any of the two comment lines are deleted. 

The error message is very odd as it doesn't even show a line number and file. Yet, I am sure the error originates here.

My Rule

![My Rule](https://github.com/snakemake/snakefmt/assets/10867778/75e0d04e-e6c5-4f27-aaed-48fd60f4d986)

The error message

![The Error](https://github.com/snakemake/snakefmt/assets/10867778/c48651be-d7cb-4360-a880-192eae141bf8)

**Comment by mbhall88** [comment]
Yes you're probably correct with the source of the error. This response https://github.com/snakemake/snakefmt/issues/197#issuecomment-1647105838 also applies here.
